### PR TITLE
First batch of navigation for change-group-rep

### PR DESCRIPTION
**Pull request by lmcewanhmrc**

### CODE REVIEW

File: app/views/changeGroupLead/NewGroupLeadCheckYourAnswerView.scala.html [hunk 1]
```diff
@@ -57,8 +57,5 @@
         id = Some("sent-your-request")
     )
     @paragraphBody(messages("newGroupLeadCheckYourAnswers.body"))
-    @linkButton(
-        "newGroupLeadCheckYourAnswers.button",
-        controllers.changeGroupLead.routes.NewGroupLeadCheckYourAnswerController.onSubmit.url
-    )
+    @linkButton("newGroupLeadCheckYourAnswers.button", call.url)
```
Comment (liampan) [inline app/views/changeGroupLead/NewGroupLeadCheckYourAnswerView.scala.html:60]: does this work? endpoint a POST? a tag is only GET ? 😕

Comment (liampan) [reply app/views/changeGroupLead/NewGroupLeadCheckYourAnswerView.scala.html:60]: also wonder if it should be a parameter, it never changes

File: app/navigation/ChangeGroupLeadNavigator.scala [hunk 2]
```diff
@@ -16,20 +16,28 @@
...
-    )
+  def enterContactAddress(implicit mode: Mode): Call = 
+    returnToCheckYourAnswersOr (routes.MainContactNameController.onPageLoad)
 
+  def mainContactName(implicit mode: Mode): Call =
+    returnToCheckYourAnswersOr (_ => routes.NewGroupLeadCheckYourAnswerController.onPageLoad) // todo
+
+  def checkYourAnswers: Call = routes.NewGroupLeadConfirmationController.onPageLoad
+
+  private def returnToCheckYourAnswersOr(call: Mode => Call)(implicit mode: Mode): Call =
```
Comment (liampan) [inline app/navigation/ChangeGroupLeadNavigator.scala:37]: nice

**Review by liampan — APPROVED** [review]
I dont think the submit on CYA will work, but doesnt look like it would have in the begining either so can fix in a future PR.

**Review by mauro-fanton — APPROVED** [review]
the changeGroupLead directory in the A11y test is duplicate (different spelling). My fault, I did spelled it wrong. I can fixed later if you just want to merge this PR.

**Pull request merged**